native-maven-plugin/src/main/java/org/graalvm/buildtools/utils/NativeImageConfigurationUtils.java

```diff
@@ -128,8 +128,8 @@ public static Path getNativeImage(Logger logger) throws MojoExecutionException {
         }
 
         if (nativeImage == null) {
-            throw new RuntimeException("GraalVM native-image is missing on your system. " + System.lineSeparator() +
-                    "Make sure that GRAALVM_HOME environment variable is present.");
+            throw new RuntimeException("The 'native-image' tool was not found on your system. " + System.lineSeparator() +
```

Comment:
Minor consistency nit, maybe remove the newline:
```suggestion
            throw new RuntimeException("The 'native-image' tool was not found on your system. " +
```